### PR TITLE
blush 0.6.0 (new formula)

### PR DESCRIPTION
**Pull request by chenrui333**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

**Comment by chenrui333** [comment]
give linux test another run

File: Formula/blush.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args(ldflags: "-s -w -X main.version=#{version}")
+  end
+
+  test do
+    # Display issue on CI
+    return if OS.linux? && ENV["HOMEBREW_GITHUB_ACTIONS"]
```
Comment (SMillerDev) [inline Formula/blush.rb:17]: Why does it need a display if it's just grep?

Comment (SMillerDev) [reply Formula/blush.rb:17]: Should probably fix the failure instead

Comment (chenrui333) [reply Formula/blush.rb:17]: yeah, I might need some help on this. cc @cho-m @danielnachun 🙏

**Pull request closed without merging**